etc/schema/mysql.schema.sql

```diff
@@ -40,3 +40,16 @@ CREATE TABLE `icingaweb_user_preference`(
   `mtime`    timestamp NULL DEFAULT NULL ON UPDATE CURRENT_TIMESTAMP,
   PRIMARY KEY (`username`,`section`,`name`)
 ) ENGINE=InnoDB DEFAULT CHARSET=utf8;
+
+CREATE TABLE rememberme (
```

Comment:
All the other tables also use a table prefix. Please add `icingaweb_` here as well.